src/kv/tx.h

```diff
@@ -388,7 +389,13 @@ namespace kv
       auto map = all_changes.begin()->second.map;
       auto e = map->get_store()->get_encryptor();
 
-      KvStoreSerialiser replicated_serialiser(e, version);
+      kv::Version mcv = max_conflict_version;
```

Comment:
Why create a distinct mcv? I thought it might be to avoid writing to an argument, but we do that at 298, so might as well set max_conflict_version again here?